coruscant/src/components/user-settings-tabs/InvitesSettings.js

```diff
@@ -30,82 +32,95 @@ const styles = {
   }
 }
 
-const tableData = [
-  {
-    id: 'abc1',
-    projectName: 'drumitdown',
-    expirationDate: '2018-03-01',
-    status: 'Open',
-  },
-  {
-    id: 'abc2',
-    projectName: 'tmnt',
-    expirationDate: '2018-01-23',
-    status: 'Expired',
-  },
-  {
-    id: 'abc3',
-    projectName: 'hammer-io',
-    expirationDate: '2018-02-18',
-    status: 'Accepted',
-  }
-]
+function getExpirationDate(invite) {
```

Comment:
could you change these to function expressions - eg: 
```
const getExpiration = (invite) => {
  // blah blah blah
}
```

Reply:
Function expressions are not hoisted, which forces you to declare it at the top, before it is used.

Reply:
I can... but what's wrong with taking advantage of hoisting?